<ui version="4.0" >
 <class>KGameCardSelectorBase</class>
 <widget class="QWidget" name="KGameCardSelectorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>521</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>620</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>1024</width>
    <height>800</height>
   </size>
  </property>
  <widget class="QFrame" name="frame_3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>602</width>
     <height>212</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QListWidget" name="frontList" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="resizeMode" >
       <enum>QListView::Adjust</enum>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="viewMode" >
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>16</width>
        <height>62</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" >
     <widget class="QListWidget" name="backList" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="resizeMode" >
       <enum>QListView::Adjust</enum>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="viewMode" >
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>230</y>
     <width>601</width>
     <height>86</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Filter</string>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="checkBoxSVG" >
        <property name="text" >
         <string>Show only scalable cards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPNG" >
        <property name="text" >
         <string>Show only fixed size cards</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="checkBoxLock" >
        <property name="text" >
         <string>Lock backside to frontside</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>44</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>322</y>
     <width>602</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Frontside</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="frontPreview" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Details</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="cardName" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>250</width>
           <height>10</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cardDescription" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Backside</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="backPreview" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
